crates/bevy_text/src/text.rs

```diff
@@ -18,16 +18,6 @@ pub struct Text {
     pub linebreak_behavior: BreakLineOn,
 }
 
-impl Default for Text {
-    fn default() -> Self {
-        Self {
-            sections: Default::default(),
-            justify: JustifyText::Left,
-            linebreak_behavior: BreakLineOn::WordBoundary,
-        }
-    }
-}
```

Comment:
`BreakLineOn::WordBoundary` is now the `Default` value, so `impl Default for Text` can be `derive`d automatically